db_cost_aws-bs.md

```diff
@@ -211,13 +210,13 @@ users-redis             ClusterIP      10.100.3.12      <none>
 Portworx is set up with a spec that is generated through a UI located at [PX-Central](https://central.portworx.com/dashboard). 
 
 The configuration is set up with the following parameters:
-1. PWX Hosted etcd - this allows portworx to run a highly available version of etcd for the cluster increasing the availability of the cluster
+1. PWX Internally Hosted etcd - this allows Portworx to run a highly available version of etcd for the cluster increasing the availability of the cluster
 2. AWS is selected as the cloud
 3. GP2 based EBS volume types selected with 200GB capacity
 
-This will bring up 2 clusters of 3 Volumes each supporting the following:
-1. 3 redundant volumes in a cluster to support Postgres @ 200GB per volume
-2. 3 redundant volumes in a cluster to support etcd for the cluster @ 150GB per volume
+This will bring up a single Portworx cluster EBS in the following ways:
+1. 3 redundant volumes (1 per EKS node) which Portworx pools together into a single virtual storage pool. Postgres uses this pool to create [PVs](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for persistence.
+2. 3 redundant volumes in the cluster to support a 3 node etcd cluster @ 150GB per volume for Portworx metadata.
 
```

Comment:
Just added clarification to the above statements and where people can learn more.